examples/complex-animation/pages/about.js

```diff
@@ -1,18 +1,23 @@
 import React from 'react'
 import Link from 'next/link'
 
-const About = () => (
-  <div className="container-fluid bg-success page">
-    <h1>About us</h1>
-    <Link href="/">
-      <a className="btn btn-light">Go back home</a>
-    </Link>
-    <style jsx>{`
-      .page {
-        height: 100vh;
-      }
-    `}</style>
-  </div>
-)
+class About extends React.PureComponent {
```

Comment:
Can you explain why you changed this from a functional component?

Reply:
The functional component always re-render when parent component re-render, so you should use PureComponent or React.memo